apps/recovery-portal/src/main/webapp/self-registration-complete.jsp

```diff
@@ -22,11 +22,24 @@
 <%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointUtil" %>
 <%@ page import="java.net.MalformedURLException" %>
 <%@ page import="java.io.File" %>
+<%@ page import="org.wso2.carbon.identity.recovery.util.Utils" %>
+<%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="java.util.HashMap" %>
+<%@ page import="java.util.Map" %>
 <jsp:directive.include file="includes/localize.jsp"/>
 <jsp:directive.include file="tenant-resolve.jsp"/>
 <%
     boolean isEmailNotificationEnabled = false;
     String callback = (String) request.getAttribute("callback");
+    String username = request.getParameter("username");
+    String userStoreDomain = request.getParameter("userstoredomain");
+    String sessionDataKey = null;
+    String fullyQualifiedUsername = username;
+
+    boolean isAutoLoginEnable = Boolean.parseBoolean(Utils.getConnectorConfig("SelfRegistration.AutoLogin.Enable",
```

Comment:
do we need to check this ? if the auto login is disabled, the cookie won't be created in the previous page right ?

Reply:
yes. It is a redundant one. I will fix it